### PR TITLE
Add vnodes to components part 10

### PR DESCRIPTION
**Pull request by affordances**

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/components/collapsible_body_text.tsx [hunk 1]
```diff
@@ -12,57 +12,46 @@ import User from './widgets/user';
 const QUILL_PROPOSAL_LINES_CUTOFF_LENGTH = 50;
 const MARKDOWN_PROPOSAL_LINES_CUTOFF_LENGTH = 70;
 
-export const formatBody = (vnode, updateCollapse) => {
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/views/components/collapsible_body_text.tsx:15]: How were we able to remove this function?

Comment (affordances) [reply packages/commonwealth/client/scripts/views/components/collapsible_body_text.tsx:15]: I took the part out that was needed and deleted the rest. I basically just DRYed up the file.

Comment (jnaviask) [reply packages/commonwealth/client/scripts/views/components/collapsible_body_text.tsx:15]: The change just assumes `item.body` is accurate -- but it looks like in some cases we will use `item.text` instead. Is this a problem?

Comment (affordances) [reply packages/commonwealth/client/scripts/views/components/collapsible_body_text.tsx:15]: The old version also handled comments, which this one doesn't, so no `text`.

Comment (zakhap) [reply packages/commonwealth/client/scripts/views/components/collapsible_body_text.tsx:15]: fine

**Review by zakhap — APPROVED** [review]

**Pull request merged**